fedot/api/main.py

```diff
@@ -23,7 +23,15 @@
 
 
 def default_evo_params(problem):
-    """ Dictionary with default parameters for composer """
+    """ Dictionary with default parameters for composer which is as given below.
+        params = {'max_depth': 3,
```

Comment:
Perhaps it would be better to provide a link to a specific method here instead of "hardcoding" params dict
You can use this [fine_tune_all_nodes](https://github.com/nccr-itmo/FEDOT/blob/master/fedot/core/pipelines/pipeline.py#L241) method as example